### DISCUSSION TITLE
Dark Mode - Reports

### DISCUSSION THREAD

**Discussion by thebaaron42** [burkybang/SuiteAdvanced-Feedback#64] [Ideas]
First off, love this extension, if only for the dark mode capabilities.  But so many good features.

When looking at a report on dark mode, it's very difficult to see the left column

Image attached.

Thanks!
Ben
<img width="607" alt="Screen Shot 2022-11-08 at 2 48 28 PM" src="https://user-images.githubusercontent.com/131487/200692748-ffdda2e1-74e4-4392-addc-41af5ee80e00.png">

**Comment by burkybang** [comment]
Thanks for reporting this! I've already resolved it, and it will be in the next release. For some reason Google is taking a very long time to approve updates lately. (Usually a day. So far over a week)

**Reply by thebaaron42** [reply]
Awesome.  Looking forward.

**Reply by thebaaron42** [reply]
Any chance this new release will come out soon?  Reports are incredibly difficult to look at.

**Reply by burkybang** [reply]
Sorry, Google took 3 weeks to approve an update to my beta testers. Usually they only take a few minutes to a few hours. Afterwards, I fixed a few bugs. I'll be submitting the production release sometime this week. Then we'll wait for Google to approve that one. Hopefully it's a quick approval this time.

Something to note about the next update: It's a big one. There are a lot of new features, bug fixes, and compatibility fixes for NetSuite 2022.2.

Most importantly, there is one permission change that will require users to accept in order to continue using SuiteAdvanced. The new permission allows SuiteAdvanced to run on the new SuiteAnswers website since NetSuite changed the domain for it. Anytime a Chrome extension has a permission change, the extension gets disabled until the user accepts the new permission. It's not always obvious why the extension stops working.

**Reply by burkybang** [reply]
The update is out now!
